frontend/src/modules/editor/components/Editor.js

```diff
@@ -162,6 +171,12 @@ export class EditorBase extends React.Component<InternalProps> {
                     sendTranslation={ this.sendTranslation }
                     updateTranslationStatus={ this.updateTranslationStatus }
                 />
+                <UnsavedChanges
+                    callback={ this.props.editor.unsavedChangesCallback }
+                    hide={ this.hideUnsavedChanges }
+                    ignore={ this.ignoreUnsavedChanges }
+                    show={ this.props.editor.unsavedChanges }
```

Comment:
This sounds like it's an action when it's actually data. How about calling it something like `changes` instead?